bin/subzero-standalone/cli/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "subzero-standalone-cli"
-version = "3.1.61"
+version = "3.1.62"
```

```suggestion
version = "3.2.62"
```
We decided on changing the minor version after the Substrate version change.